README.md

```diff
@@ -239,7 +239,7 @@ Rust as a first-party feature.
 
 By default, Ruff enables Flake8's `E` and `F` rules. Ruff supports all rules from the `F` category,
 and a [subset](https://docs.astral.sh/ruff/rules/#error-e) of the `E` category, omitting those
-stylistic rules made obsolete by the use of an autoformatter, like
+stylistic rules made obsolete by the use of a formatter, like
```

Comment:
Hey we already say we won't include these

Reply:
The first and second sentence in the paragraph sound contradictory to me

Reply:
How so?